web/src/OverviewResourceSidebarTypeFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import styled from "styled-components"
+import { ResourceFilters } from "./types"
+
+const SidebarResourceTypeFilterStyle = styled.div`
```

Comment:
instead of using css to hide these, can we just not include the element at all?

tho i guess this is all going away soon anyway.

Reply:
Yes def can do it with a conditional--tho what do you mean "this is all going away soon"?